### ISSUE TITLE
Histogram for Throughput inaccessible

### ISSUE THREAD

**Issue by Earthnuker** [magnet/metered-rs#30]
Hello,

according to #26 the inner histogram of metrics should be exposed, however this seems to not be the case for the `Throughput` metric (unless i'm doing something wrong)

i've put the relevant parts of my code into a gist at https://gist.github.com/Earthnuker/87de2d8e40913b2e20d26e1af24a480a

Best regards,

Earthnuker

**Comment by magnet** [comment]
Hi @Earthnuker 

Thanks for your bug report.  I have fixed this in `3b1d92c` and will release shortly.  Here's your updated snippet:

```rust

pub struct Router {
    metrics: RouterMetrics
}

#[metered(registry = RouterMetrics, visibility = pub )]
impl Router {
    pub fn new() -> Self {
        Self {
            metrics: RouterMetrics::default()
        }
    }

    #[measure([HitCount, Throughput])]
    fn foo(&self) -> usize {
        42
    }

    fn this_works_but_clones_the_histogram(&self) -> () {
        let tp=&self.metrics.foo.throughput;
        println!("{}±{}/s",tp.0.histogram().mean(),tp.0.histogram().stdev());
    }

    fn this_works_and_doesnt_clones(&self) -> () {
        let tp=&self.metrics.foo.throughput;
        let guard = tp.inner.lock();
        println!("{}±{}/s",guard.hdr_histogram.mean(),guard.hdr_histogram.stdev());
    }
}
```

**Comment by magnet** [comment]
This is fixed in metered 0.7.0
